styles/_base.css

```diff
@@ -0,0 +1,98 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  /* border: 1px solid red; */
+}
+
+a,
+a:hover,
+a:focus,
+a:active {
+  text-decoration: none;
+  color: inherit;
+}
+
+body {
+  background-color: white;
+  color: var(--primary);
+  font-family: sans-serif;
+  font-size: 1.3125rem;
+  font-weight: 100;
+  line-height: 1.64;
+  color: var(--black);
+}
+
+body,
+h1,
+h2,
+h3,
+p {
+  margin: 0;
+}
+
+h1,
+h2,
+h3 {
+  line-height: 1;
+}
+
+section,
+address {
+  padding: 3rem 0;
+}
+
+h2,
+h3 {
+  margin-bottom: 1em;
+}
+
+input,
+button,
+textarea {
+  font-size: inherit;
+}
+
+h2 {
+  font-size: 2.5rem;
+  font-weight: 600;
+  color: var(--headings);
+}
+
+h3 {
+  font-size: 1.6rem;
+  font-weight: 400;
+  color: var(--headings);
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+nav {
```

Comment:
This should be in the styles for the header as it is very specific. Would probably have a class name with `header__nav`